src/backends/beamcoder.ts

```diff
@@ -256,12 +255,12 @@ export class BeamcoderExtractor extends BaseExtractor implements Extractor {
             return null;
         }
         const rawData = this._resizeFrameData(frame);
-        const image = createImageData(
-            rawData,
-            frame.width,
-            frame.height
-        ) as ImageData;
-        return image;
```

Comment:
It was nice to have ImageData type, but since it comes with complex build requirements, I think it makes sense to remove 😿